test/functional/app/api/test_ussd_api.py

```diff
@@ -0,0 +1,137 @@
+import pytest
+from faker.providers import phone_number
+from faker import Faker
+from functools import partial
+
+from helpers.ussd_utils import create_transfer_account_for_user, make_kenyan_phone
+from server import db
+from helpers.user import UserFactory
+from server.models.token import Token
+from server.models.user import User
+from server.models.ussd import UssdSession, UssdMenu
+from server.utils.user import default_transfer_account
+
+fake = Faker()
+fake.add_provider(phone_number)
+phone = partial(fake.msisdn)
+
+
+@pytest.fixture(scope='module')
+def init_menus(test_client, init_database):
```

Comment:
is it possible to run seed.py in test env? then we can drop this

Reply:
ummmmm we would import it, and then call the methods?